sdk/aqueduct/dag.py

```diff
@@ -50,10 +52,23 @@ class AirflowEngineConfig(BaseModel):
     integration_id: uuid.UUID
 
 
+class K8sEngineConfig(BaseModel):
+    integration_id: uuid.UUID
+
+
 class EngineConfig(BaseModel):
     type: RuntimeType = RuntimeType.AQUEDUCT
     aqueduct_config: Optional[AqueductEngineConfig]
     airflow_config: Optional[AirflowEngineConfig]
+    k8s_config: Optional[K8sEngineConfig]
+
+
+class FlowConfig(BaseModel):
```

Comment:
I thought it made sense to leave this in `flow.py` since its technically the Flow config that a user specifies, whereas `Dag` and `EngineConfig` are more internal concepts that the user doesn't need to know about.

Reply:
I'm actually curious why these config classes are in the dag file to begin with? EngineConfigs seem like a separate concept from the dag, which is more structural info than runtime info.